pmd-core/src/main/java/net/sourceforge/pmd/lang/document/TextFileContent.java

```diff
@@ -100,6 +100,18 @@ public long getCheckSum() {
         return checkSum;
     }
 
+    /**
+     * Converts a line/column pair to a character offset.
+     *
+     * @param line   Line number (1-based)
+     * @param column Column number (1-based)
+     *
+     * @return Text offset (zero-based), or -1
+     */
+    public int offsetFromLineColumn(int line, int column) {
```

Comment:
If this stays public API, consider using TextPos2d

Reply:
Fixed in 17601960a3fb7fe8dc7a0158f6aa73eba6b7cb70 - moved this method to TextDocument and refactored the usages.